src/components/Icon/BankIcons.js

```diff
@@ -0,0 +1,115 @@
+import DefaultBank from '../../../assets/images/bank.svg';
+import AmericanExpress from '../../../assets/images/bankicons/american-express.svg';
+import BankOfAmerica from '../../../assets/images/bankicons/bank-of-america.svg';
+import BB_T from '../../../assets/images/bankicons/bb-t.svg';
+import CapitalOne from '../../../assets/images/bankicons/capital-one.svg';
+import CharlesSchwab from '../../../assets/images/bankicons/charles-schwab.svg';
+import Chase from '../../../assets/images/bankicons/chase.svg';
+import CitiBank from '../../../assets/images/bankicons/citibank.svg';
+import CitizensBank from '../../../assets/images/bankicons/citizens-bank.svg';
+import Discover from '../../../assets/images/bankicons/discover.svg';
+import Fidelity from '../../../assets/images/bankicons/fidelity.svg';
+import HuntingtonBank from '../../../assets/images/bankicons/huntington-bank.svg';
+import NavyFederalCreditUnion from '../../../assets/images/bankicons/navy-federal-credit-union.svg';
+import PNC from '../../../assets/images/bankicons/pnc.svg';
+import RegionsBank from '../../../assets/images/bankicons/regions-bank.svg';
+import SunTrust from '../../../assets/images/bankicons/suntrust.svg';
+import TdBank from '../../../assets/images/bankicons/td-bank.svg';
+import USBank from '../../../assets/images/bankicons/us-bank.svg';
+import USAA from '../../../assets/images/bankicons/usaa.svg';
+
+/**
+ * Returns matching asset icon for bankName
+ * @param {String} bankName
+ * @returns
+ */
+
+function getAssetIcon(bankName) {
+    if (bankName.includes('americanexpress')) {
+        return AmericanExpress;
+    }
+
+    if (bankName.includes('bank of america') || bankName.includes('bankofamerica')) {
+        return BankOfAmerica;
+    }
+
+    if (bankName.startsWith('bbt')) {
+        return BB_T;
+    }
+
+    if (bankName.startsWith('capital one') || bankName.includes('capitalone')) {
+        return CapitalOne;
+    }
+
+    if (bankName.startsWith('chase') || bankName.includes('chase')) {
+        return Chase;
+    }
+
+    if (bankName.includes('charles schwab') || bankName.includes('charlesschwab')) {
+        return CharlesSchwab;
+    }
+
+    if (bankName.startsWith('citibank') || bankName.includes('citibank')) {
+        return CitiBank;
+    }
+
+    if (bankName.startsWith('citizens bank') || bankName.includes('citizensbank')) {
+        return CitizensBank;
+    }
+
+    if (bankName.startsWith('discover ') || bankName.includes('discover.') || bankName === 'discover') {
```

Comment:
Wouldn't `.includes('discover')` cover all 3 cases here (and in the other similar ones)?

Reply:
There's a bank Discovery